tests/Behat/Context/Application/RefundingContext.php

```diff
@@ -258,7 +265,7 @@ public function shouldStillBeAbleToRefundOrderShipment(PaymentMethodInterface $p
             $this->commandBus->dispatch(new RefundUnits(
                 $this->order->getNumber(), [], [new ShipmentRefund($shippingAdjustment->getId(), $remainingTotal)], $paymentMethod->getId(), ''
             ));
-        } catch (CommandDispatchException $exception) {
+        } catch (\Exception $exception) {
```

Comment:
Can we typehint for a more specific exception? So that we can be sure that our application failed, not the infrastructure or anything else?